server/test/suites/categoryRouter.test.js

```diff
@@ -25,85 +25,87 @@ let testUser2;
 let testCategory1;
 let testCategory2;
 
-before(async () => {
-  // Connect to in-memory mongodb
-  mongoServer = new MongoMemoryServer();
-  const mongoUri = await mongoServer.getUri();
-  getMongoDB(mongoUri);
-
-  // Populate the in memory database
-  try {
-    testUser1 = new User({ email: 'lobster1@gmile.com' });
-    await testUser1.save();
-
-    testUser2 = new User({ email: 'lobster2@gmile.com' });
-    await testUser2.save();
+module.exports =  function categorySuite() {
+  before(async () => {
+    // Connect to in-memory mongodb
+    mongoServer = new MongoMemoryServer();
+    const mongoUri = await mongoServer.getUri();
+    getMongoDB(mongoUri);
+  
+    // Populate the in memory database
+    try {
+      testUser1 = new User({ email: 'lobster1@gmile.com' });
+      await testUser1.save();
+  
+      testUser2 = new User({ email: 'lobster2@gmile.com' });
+      await testUser2.save();
+  
+      testCategory1 = new Category({
+        title: 'User1 Category1',
+        userId: testUser1._id,
+      });
+      
+      testCategory2 = new Category({
+        title: 'User2 Category1',
+        userId: testUser2._id,
+      });
+      
+      item1 = new Item({
+        sourceLink: 'www.googe.com', 
+        placesId: 'something', 
+        userId: testUser1._id, 
+        highlight: 'highlight words',
+        comment: 'sample comments',
+        categoryIds: [testCategory1._id],
+      });
+      await item1.save();
+  
+      item2 = new Item({
+        sourceLink: 'www.googe.com', 
+        placesId: 'something else', 
+        userId: testUser1._id, 
+        highlight: 'other highlighted words',
+        comment: 'another sample comment',
+        categoryIds: [testCategory1._id],
+      });
+      await item2.save();
+  
+      testCategory1.items = [item1._id, item2._id];
+      await testCategory1.save();
+      await testCategory2.save();
+  
+    } catch (error) {
+      expect(error).to.be.null;
```

Comment:
Just a reminder that this (and any similar setup) should be changed per @ricazhang's request

Reply:
Yep, will make changes to this part